src/tests/ftest/pool/create_test.yaml

```diff
@@ -23,7 +23,16 @@ server_config:
       scm_class: dcpm
       scm_list: ["/dev/pmem0"]
       scm_mount: /mnt/daos0
-pool:
+pool_1:
   name: daos_server
   control_method: dmg
-  quantity: 200
+  scm_size: 90%
+  svcn: 1
```

Comment:
This should be the default and no need to pass it?

Reply:
The default for the `svcn` TestPool parameter is None.  This setting ensures that `--nsvc=1` is included in the `dmg pool create` command.